src/definitions/j.json

```diff
@@ -0,0 +1,49 @@
+[
+    {
+      "display": "justify-content",
```

Comment:
This will be a display name so let it be as friendly as possible. something like 
```
Vertical Align Content
```

or just 

```
Justify Content
```